okapi-core/src/main/java/org/folio/okapi/MainVerticle.java

```diff
@@ -337,8 +342,16 @@ public void start(Promise<Void> promise) {
   }
 
   private Future<Void> startRedeploy() {
-    return discoveryManager.restartModules()
-        .compose(res -> tenantManager.startTimers(discoveryManager, okapiVersion));
```

Comment:
Is the idea to maintain changed timers? If yes, is there an easy to reset to default MD timers?

Reply:
Yes. The changed timers are stored.

I'll make a call to reset a timer (to original value).

Reply:
This is now done.